### ISSUE TITLE
Better icon for zxtns with no memo

### ISSUE THREAD

**Issue by leto** [MyHush/SilentDragon#145]
Currently it's a white box that almost looks like a broken image.

**Comment by csharpee** [comment]
@DenioD I remember you having a icon list, but am unable to find it.

This is also related to #147 


For future aid, 

L161-178 via txtablemodel.cpp gives an example of adding icons.

**Comment by leto** [comment]
@csharpee yeah, I think @DenioD did something related to this but I am not sure where the code is
